### ISSUE TITLE
Include instead of Exclude playlist?

### ISSUE THREAD

**Issue by Delaser** [jbh-cloud/spotify_sync#29]
Looking for a way to only specificy a specific playlist to download, rather than excluding ones I don't want.

I like to have individual directories for my music. Would make it much easier.

Thanks!

**Comment by fabifuu** [comment]
Yeah, I also thought is there a way to download a specific playlist and make into a single folder (not categorized into artist's folder)

**Comment by sacnoth** [comment]
Having both would be really great.  
Including specific albums or playlists by their ID.

I'd love to download playlists like "Your top songs 2023", but as I'm not the owner of the playlist according to Spotify `spotify_sync` won't download the playlist.

My current workaround is to copy all songs of an album or playlist into a new playlist so `spotify_sync` will pick it up.
